video/out/drm_common.c

```diff
@@ -531,11 +597,19 @@ struct kms *kms_create(struct mp_log *log, const char *connector_spec,
     int card_no = -1;
     char *connector_name = NULL;
     parse_connector_spec(log, connector_spec, &card_no, &connector_name);
+    char *primary_node_path = get_primary_device_path(log, &card_no);
```

Comment:
never freed?

Reply:
Yup, good notice. I was really focusing too quickly on getting the next commit done, and then cleaning it up :)